bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java

```diff
@@ -81,6 +83,8 @@
     private String unitLabel;
 
     private String context;
+
+    @SerializedName("default")
```

Comment:
IMHO we should use DTOs for (de)serialization over the REST interface.
All the time.
Without DTOs we could export internal stuff.